confgenerator/testdata/invalid/linux/metrics-receiver_unsupported_type/golden_error

```diff
@@ -1 +1 @@
-metrics receiver with type "unsupported_type" is not supported. Supported metrics receiver types: [hostmetrics, jvm, nginx].
```

Comment:
Any Cassandra specific invalid config input we want to test against?

One potential item is that the current implementation does not support more than one JVM based receivers as discussed in https://github.com/GoogleCloudPlatform/ops-agent/pull/220/files#r724704460. In other words, at most one metrics receiver with type "cassandra" or "jvm" is allowed. Adding this validation has its pros and cons though:

* Pros: Prevent users from accidentally specifying multiple JVM based receivers in the config without realizing something is wrong. If the errors generated in the logs makes it very clear this is the cause, it might not be that bad. (Is that the case?) Otherwise this would generate unnecessary support cases. Making it clear in the documentation would reduce the chance, but eliminating it.
* Cons: When we do support multiple JVM based receivers, we would need to do work to remove this validation.

Reference for similar validation:
* https://github.com/GoogleCloudPlatform/ops-agent/blob/8bd47ae99fbf38e6b54824cb4dcb0de7cb5961d1/confgenerator/config.go#L613
* https://github.com/GoogleCloudPlatform/ops-agent/blob/master/confgenerator/testdata/invalid/windows/metrics-receiver_more_than_one_with_type_iis/golden_error

If the effort to add this validation is relatively low, I'd prefer we add a validation to prevent users from trying out something that is not supported and wasting the effort.

WDYT?